webhookValidationConfig.go

```diff
@@ -60,3 +72,30 @@ func (config *ValidatorConfig) BuildURLChecker() (*urlegit.Checker, error) {
 	}
 	return urlegit.New(o...)
 }
+
+// BuildOptions translates the configuration into a list of options to be used to validate the registration
+func (config *ValidatorConfig) BuildOptions(checker *urlegit.Checker) []webhook.Option {
+	var opts []webhook.Option
+	if config.Opts.AtLeastOneEvent {
+		opts = append(opts, webhook.AtLeastOneEvent())
+	}
+	if config.Opts.EventRegexMustCompile {
+		opts = append(opts, webhook.EventRegexMustCompile())
+	}
+	if config.Opts.DeviceIDRegexMustCompile {
+		opts = append(opts, webhook.DeviceIDRegexMustCompile())
+	}
+	if config.Opts.ValidateRegistrationDuration {
```

Comment:
`ValidateRegistrationDuration` option is very similar to the old ancla options `CheckUntil` `CheckUntilOrDurationExist` and `CheckDuration` however, `CheckUntil` includes `jitter` and `now func` which `ValidateRegistrationDuration` does not - do we want to edit `ValidateRegistrationDuration` to include `jitter` and `now func`

Reply:
ancla validator: https://github.com/xmidt-org/ancla/blob/e805e4b4ede085fb97b4fcab8ca7a1ebc2216520/webhookValidator.go#L113

webhook option: https://github.com/xmidt-org/webhook-schema/blob/0ee06ff248b57a80fc2aa72050df28de3f520b24/webhook.go#L266 (this is just for registrationv1 - this is same code copied from: https://github.com/xmidt-org/webhook-schema/blob/6d8185efa55bea9cb1fb3af6897468011f25ed66/options.go#L110)